agents/mysql/perfschema/perfschema_test.go

```diff
@@ -453,4 +453,19 @@ func TestPerfSchema(t *testing.T) {
 		require.NoError(t, err)
 		tests.LogTable(t, structs)
 	})
+
+	t.Run("DisableQueryExamples", func(t *testing.T) {
+		m := setup(t, db, true)
```

Comment:
That's the problem with positional boolean arguments: does this `true` means disable or enable – and then disable or enable _what_ exactly? You can't tell without going to `setup` function implementation and then hoping to `newPerfSchema`. That's why positional boolean arguments are often considered a bad practice.

There are one and a half typical solutions in Go:
1. Use custom type and typed constants with good names. Unfortunately, due to [how underlying types and assignability works](https://golang.org/ref/spec#Assignability), that will not prevent one from using `true`/`false` directly: https://goplay.space/#hMnfuzpwZ-T. So that's only half of the solution.
2. Use params/options structures. As a bonus point, that often adds useful zero values.

We already have Params struct there, so let's use the same approach and introduce a new type (`newParams`?) that will contain all `newPerfSchema` parameters.